### PR TITLE
feat(tests): adding test coverage reports

### PR DESCRIPTION
**Pull request by milos1290**

### CODE REVIEW

File: AndroidSDK/src/main/java/com/leanplum/LeanplumPushService.java [hunk 1]
```diff
@@ -35,6 +35,7 @@
 import android.os.Bundle;
 import android.support.v4.app.NotificationCompat;
 import android.text.TextUtils;
+import android.widget.RemoteViews;
```
Comment (Enuviel) [inline AndroidSDK/src/main/java/com/leanplum/LeanplumPushService.java]: I think this one not from this code change.

File: AndroidSDK/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">AndroidSDK</string>
```
Comment (Enuviel) [inline AndroidSDK/src/main/res/values/strings.xml]: do we use this one only for tests?

Comment (milos1290) [reply AndroidSDK/src/main/res/values/strings.xml]: we don't actually use it anywhere.

File: AndroidSDK/src/main/java/com/leanplum/LeanplumPushService.java (hunk 1 above)
Comment (milos1290) [reply AndroidSDK/src/main/java/com/leanplum/LeanplumPushService.java]: yeah it looks like. no idea how it got through.

**Comment by Enuviel** [comment]
test this please

**Comment by Enuviel** [comment]
Blocked by https://github.com/powermock/powermock/issues/727

**Pull request closed without merging**